### ISSUE TITLE
Open Source Friday - Supabase - [11-08-2024]

### ISSUE THREAD

**Issue by thorwebdev** [githubevents/open-source-friday#97]
### Name

Thor Schaeff

### GitHub Handle

@thorwebdev

### Tell us about yourself

I'm a Developer Experience Engineer at Supabase, working on client libraries, docs, examples, and most recently VS Code and GitHub Copilot extensions to make the development experience with Supabase as delightful as possible. The entire Supabase Stack is open source, and so is the extension code as well. Would love to show folks the open secret sauce :) 

### Project Name

Supabase VS Code Extension

### Project Repo Link

https://github.com/supabase-community/supabase-vscode-extension

### Stream Date

- [X] Yes
- [ ] Not yet

### Dates

11-08-2024

### Twitter URL

@thorwebdev

### LinkedIn URL

https://www.linkedin.com/in/thorwebdev/

**Comment by LadyKerr** [comment]
Hey @thorwebdev ! We'd love to have you come on to our show! We're still waiting to start that new show BUT we'd also love to have you come on and chat about your amazing project.

If you're still interested, please choose a date from this cal: https://gh.io/osf-booking 

Let me know which date you choose, if you decide to!

**Comment by thorwebdev** [comment]
Thanks @LadyKerr. I've since moved on from Supabase, but I think @dshukertjr would be a great fit for this!

**Comment by dshukertjr** [comment]
Hi @LadyKerr ! I would love to take this one if it's alright. Also about the topic, could we talk about various open source technologies we create/ use at Supabase instead of the originally proposed Supabase VS Code extension?

Twitter URL
@dshukertjr

LinkedIn URL
https://www.linkedin.com/in/dshukertjr/

**Comment by AndreaGriffiths11** [comment]
Sounds great @dshukertjr If you're still interested, please choose a date from this cal: https://gh.io/osf-booking

**Comment by LadyKerr** [comment]
Hey @dshukertjr we can actually just talk about Supabase since its opensource? Then we can also weave in what open source projects its built upon!

Also, choose a date and let us know what date you choose?

**Comment by dshukertjr** [comment]
Hi @AndreaGriffiths11 @LadyKerr !

I'm actually not seeing any available time slots, but what time do you have set up as the time slots for these recordings?

<img width="1405" alt="Image" src="https://github.com/user-attachments/assets/22769687-0523-4644-9f60-ece14783b15f" />

**Comment by AndreaGriffiths11** [comment]
Hi @dshukertjr the calendar is looking at your avaiability as well, might be a time zone issue if you have certain time blocks in your calendar. OSF is hosted live Fridays at 1PM ET the apoointment is set for 12:45PM ET for tech checks. For this one we might consider pre-recording at a time that is more convenient for your timezone but the show will still air at the 1PM on a Friday.

**Comment by dshukertjr** [comment]
>the calendar is looking at your avaiability as well, might be a time zone issue if you have certain time blocks in your calendar

Yup, I figured so, so I asked what time the appointment was set at 👍

If we do it at another time, what time would you like to do it? Do you have another calendar link that I could book a time outside of the designated time for the OSF appointment? Otherwise, please feel free to book a time in my calendar here: https://calendly.com/tyler-supabase/30min

**Comment by LadyKerr** [comment]
@dshukertjr try adding a time on the cal again!

Linky: https://gh.io/osf-booking

**Comment by AndreaGriffiths11** [comment]
Tyler Shukert: I'm a DevRel engineer at Supabase mainly focused on Flutter and Vibe coding content.
